.github/workflows/_build-wheel-release-upload.yml

```diff
@@ -24,6 +29,8 @@ on:
 jobs:
   tag-privilege-check:
     uses: ./.github/workflows/_release_tag_privilege_check.yml
+    with:
+      github_admin_username: ${{ inputs.github_admin_username }}
```

Comment:
then, `github_admin_username` is passed to our another reuseable `_release_tag_..yml` internally